lib/main_common.pm

```diff
@@ -854,7 +854,7 @@ sub load_inst_tests {
             loadtest "installation/sles4sap_product_installation_mode";
         }
         # Kubic doesn't have a partitioning step
-        if (!is_caasp('kubic')) {
+        unless (is_caasp) {
```

Comment:
This has nothing to do with `kubic` now ?

Reply:
is_caasp returns true for both caasp & kubic - kubic is just filter parameter
```perl
is_caasp = is_caasp('caasp') || is_caasp('kubic')
```

Reply:
ok